bond_test.go

```diff
@@ -38,6 +45,11 @@ func (a *Account) CollectionName() string {
 	return DB.Account.Name()
 }
 
+func (a Account) AfterCreate() {
```

Comment:
This `AfterCreate()` does not accept any parameter. The only way to perform a query is using a `DB` global. The problem is that this `DB` is not within the transaction context and thus it is not affected by a transaction commit or rollback. In other words, the rows created by this `AfterCreate()` will not be discarded in case of a transaction error.